deployment/network-operator/values.yaml

```diff
@@ -84,4 +84,6 @@ devicePlugin:
   # example:
   resources:
     - name: rdma_shared_device_a
-      devices: [enp5s0f0]
\ No newline at end of file
+      vendors: [15b3]
+      deviceIDs: ["1017"]
+      ifNames: [enp5s0f0]
```

Comment:
missing newline

Reply:
lets change the default to something that will be usable out of the box.
```
name: rdma_shared_device
vendors: [15b3]
```

Reply:
changed